.drone.yml

```diff
@@ -1,36 +1,148 @@
-pipeline:
-  test_node16:
+# sdk @iexec/web3mail default pipeline
+kind: pipeline
+type: docker
+name: sdk-default
+
+trigger:
+  event:
+    exclude:
+      - promote
+steps:
+  - name: install
+    image: node:16
+    pull: always
+    commands:
+      - node -v
+      - npm -v
+      - npm ci
+
+  - name: test_node16
+    image: node:16
+    pull: true
+    commands:
+      - npm ci
+      - npm run build
+      - npm test
+
+  - name: test_node14
+    image: node:14
+    pull: true
+    commands:
+      - npm ci
+      - npm run build
+      - npm test
```

Comment:
why make test e2e for both node14 and node16 ?
why not just do one test for node18 as web3mail ? 
Need to separate build and test into two different steps 
Same comment for the other pipeline below